core/modules/admin_bar/js/admin_bar.js

```diff
@@ -418,6 +418,7 @@ Backdrop.adminBar.behaviors.search = function (context, settings, $adminBar) {
     else {
       $(this).removeClass('active-search-item');
```

Comment:
```suggestion
      $this.removeClass('active-search-item');
```

Reply:
The same change could be made on line 416, GitHub won't let me comment on that line though.